src/apps/frontend/pages/header/header.tsx

```diff
@@ -0,0 +1,38 @@
+import { Block } from 'baseui/block';
+import { HeaderNavigation } from 'baseui/header-navigation';
+import * as React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { useAuthContext } from '../../contexts';
+
+import NavbarRightSection from './navbar-sections/navbar-right-section';
+
+interface HeaderNavbarProps {}
+
+const Header: React.FC<HeaderNavbarProps> = () => {
+  const navigate = useNavigate();
+  const { logout } = useAuthContext();
+
+  const handleSignOut = () => {
+    logout();
+    navigate('/login');
```

Comment:
Create a routes file instead of hardcoding the path

Reply:
Addressed